xz/plan.ps1

```diff
@@ -1,11 +1,11 @@
 $pkg_name="xz"
 $pkg_origin="core"
-$pkg_version="5.2.4"
+$pkg_version="5.2.3"
 $pkg_description="XZ Utils is free general-purpose data compression software with a high compression ratio. XZ Utils were written for POSIX-like systems, but also work on some not-so-POSIX systems. XZ Utils are the successor to LZMA Utils."
 $pkg_upstream_url="http://tukaani.org/xz/"
 $pkg_license=@("gpl2+", "lgpl2+")
```

Comment:
Could you update the licenses to be `GPL-2.0-or-later` and `LGPL-2.0-or-later`.  The `plan.sh` will also need these changes :smile:

Reply:
Fixed.

Reply:
> Could you update the licenses to be `GPL-2.0-or-later` and `LGPL-2.0-or-later`. The `plan.sh` will also need these changes 😄

Hi @smacfarlane.
* I've updated both linux and powershell plans to be 5.2.4 and to have the correct license format.
* The linux tests pass against glibc old (2.27) and new (2.29)
* The powershell tests also pass for the version upgrade to 5.2.4